ui/tests/acceptance/oidc-config/scopes-test.js

```diff
@@ -79,7 +79,7 @@ module('Acceptance | oidc-config/scopes', function (hooks) {
     assert.equal(
       currentRouteName(),
       'vault.cluster.access.oidc.scopes.scope.details',
-      'navigates to detail view after save'
+      'navigates to scopes detail view after save'
```

Comment:
added additional word here to make different from assignment acceptance test doing the same thing. That way if the test fails and we're looking up the failure message, we can more easily narrow it down.

Reply:
good call! thank you